index.html

```diff
@@ -52,7 +52,156 @@ <h2 id="me">I'am a Software Engineer</h2>
+              <p>A daily selection of privately personalized reads; 
+                 no accounts or sign-ups required.
+                 has been the industry's standard</p>
+              <ul class="used1-fr">
+                <li><a class="used1">html</a></li>
+                <li><a class="used1">Boostrap</a></li>
+                <li><a class="used1">Ruby</a></li>
+              </ul>
+              <form class="work-btn" action="#myproject">
+                <input class="see-btn" type="submit" name="button"  value="See Project"/>
+              </form>
+            </div>
+          </div>
+          <div class="card1-work">
+            <a><img src="images/project-image.png" alt="img"></a>
+            <div class="card1-work-content">
+              <h1 class="work-title">Profesional Art Printing Data</h1>
+              <p>A daily selection of privately personalized reads; 
+                 no accounts or sign-ups required.
+                 has been the industry's standard</p>
+              <ul class="used1-fr">
+                <li><a class="used1">html</a></li>
+                <li><a class="used1">Boostrap</a></li>
+                <li><a class="used1">Ruby</a></li>
+              </ul>
+              <form class="work-btn" action="#myproject">
+                <input class="see-btn" type="submit" name="button"  value="See Project"/>
+              </form>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="main-about">
+        <div class="about-me">
+            <h1 id="abt-title">About me</h1>
+          <p id="abt-text">Hello I’m a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience!
+             f you like what you see and have a project you need coded,
+             don’t hestiate to contact me.</p>
+            <form action="#myproject">
+              <input id="abt-btn" type="submit" name="button"  value="Get my Resume"/>
+            </form>
+          <img id="abt-frame" src="images/about-frame.png" alt="img"><hr id="abt-line"> 
+        </div>
+        <div class="about-categories">
+          <ul id="category">
+            <li><img src="images/icon-language.png"  alt="img"><h3 id="lang">Languages</h3>
+              <a>Javascript</a>
+              <a>Ruby</a>
+              <a>Html</a>
+              <a>Css</a>
+            </li>
+            <li><img src="images/icon-framework.png" id="fram-icon" alt="img"><h3 id="fram">Frameworks</h3>
+              <a>Ruby on Rails</a>
+              <a>RSPec</a>
+              <a>CapyBara</a>
+              <a>React</a>
+            </li>
+            <li><img src="images/icon-skills.png" id="skill-icon" alt="img"><h3 id="skill">Skills</h3>
+              <a>Codekit</a>
+              <a>Github</a>
+              <a>Codepen</a>
+              <a>Gitlab</a>
+              <a>Terminal</a>
+            </li>
```

Comment:
- [ ] Let's change a tag to li tag so we can list them according to the given guidelines.